src/main/java/racing/domain/Cars.java

```diff
@@ -0,0 +1,32 @@
+package racing.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Cars extends ArrayList<Car> {
+    private List<Car> cars;
+
+
+    static final int NAME_LENGTH = 5;
+    static final String CAR_NAME_SPLIT = ",";
+
+    public Cars initCars(String carNamesStr) {
+        String[] carNames = initCarName(carNamesStr);
+        Cars cars = new Cars();
+        for (String carName : carNames) {
+            cars.add(new Car(carName));
+        }
+        return cars;
+    }
```

Comment:
```suggestion
    public Cars(String carNamesStr) {
        this.cars = initCars(carNamesStr); 
    }

    private List<Car> initCars(String carNamesStr) {
        this.cars = new ArrayList<>();
        String[] carNames = initCarName(carNamesStr);
        for (String carName : carNames) {
            cars.add(new Car(carName));
        }
        return cars;
    }
```

생성자를 통해 초기화를 한다면 상태 값을 더 잘 활용할 수 있을 것 같아요!!